src/nccl_ofi_net.c

```diff
@@ -399,7 +401,7 @@ static inline int free_nccl_ofi_req(nccl_ofi_req_t *req, bool dec_inflight_cmds)
 	return ret;
 }
 
-static int in_list(char *item, char *list)
+static int in_list(const char *item, const char *list)
```

Comment:
Any idea why this routine duplicates the list?  strtok() doesn't require it...

Reply:
because someone wrote it that way?